packages/manager/modules/models/service/Pricing.class.js

```diff
@@ -20,6 +20,7 @@ export default class Pricing {
     return Intl.NumberFormat(convertLanguageFromOVHToBCP47(this.locale), {
       style: 'currency',
       currency: this.price.currencyCode,
+      currencyDisplay: 'narrowSymbol',
```

Comment:
FYI https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat#browser_compatibility

Time to say goodbye to Internet Explorer 11 support then? :smile: